app/src/main/java/Script/BookInfo.java

```diff
@@ -1,22 +1,27 @@
 package Script;
 
+import org.json.JSONArray;
+import org.json.JSONObject;
+
 import java.io.Serializable;
 
-public class BookInfo implements Serializable {
-	private String _projectName;
-	public String Name;
-	public String Abbr;
-	public int ChapterCount;
-	public int BookNumber;
+public class BookInfo implements Serializable
+{
+	private String projectName;
+	private String name;
+	private String abbr;
+	private int chapterCount;
+	private int bookNumber;
 
-	public IScriptProvider getScriptProvider() {
+	public IScriptProvider getScriptProvider()
+	{
```

Comment:
OTOH this is a C# standard which defies the google one we normally use for Java and JavaScript. https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.1-braces. Let's stick with no line break before the opening brace.